dataflow/src/main/java/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -303,4 +305,32 @@ public String toString() {
         String stringGraph = (String) res.get("stringGraph");
         return stringGraph == null ? super.toString() : stringGraph;
```

Comment:
If `stringGraph` is null, you could use `repr` instead of the un-helpful super implementation.

Reply:
That's a change to `toString()`, and `repr()` is more about the internal implementation.  (Also, this pull request doesn't make any changes to `toString()`.)  Should the code instead throw `BugInCF` if `stringGraph` is null?  A separate pull request could do that.